eeps/eep-0064.md

```diff
@@ -6,16 +6,16 @@
     Erlang-Version: OTP-27
     Post-History:
 ****
-EEP 64: Multi-line Indented Verbatim Strings
+EEP 64: Verbatim Multi-line Indented Strings
 ----
 
 Abstract
 ========
 
-This EEP proposes the introduction of Multi-line Indented Verbatim strings,
-*MIV Strings* and defines their semantics.  The main benefit is to allow
-*multi-line* strings in an easy and useful, *indented*, way,
-similar to in other languages, e.g Elixir.
+This EEP proposes the introduction of Verbatim Multi-line Indented strings,
+*VMI Strings*, and defines their semantics.  The main benefit is to allow
```

Comment:
@essen: I guess the fitting general feature here would be Multi-line Indented string, that has escape sequences.  The question is if that merits a prefix...

@zuiderkwast: I just wanted to call them something in the EEP that was not too long since I have to mention them many times.  "Triple quoted strings" is a better name, even if they could be "N>=3 quoted strings".  "Verbatim strings" is another possibility.  But if we later add prefixes to make them more flexible, e.g. loose "verbatim",  it is maybe "triple quoted" that is the one property that must remain...

Reply:
"Triple quoted" or "Multi-line".  I'd say we cannot take that away from them either. That they can be indented is just what makes them good for multi-line.